src/config/default.ts

```diff
@@ -35,6 +35,7 @@ export const ReviewStatus = {
 
 export const UserRoles = {
   REVIEWER: "reviewer",
+  EVALUATOR: "evaluator",
   ADMIN: "admin",
 } as const;
```

Comment:
Where is this code being used? It seems like a duplicate of the `UserRole` type

Reply:
It's being used in `src/components/tables/TableItems.tsx`
`   <MenuList color="black">
      {Object.values(UserRoles).map((role) => (
        <MenuItem
          key={role}
          textTransform={"capitalize"}
          onClick={() => {
            handleRequest(role);
          }}
        >
          {role}
        </MenuItem>
      ))}`

Reply:
We should strive to have a single source of truth. We shouldn't need to change this in multiple places when we want to add/remove a value.
We can probably derive the type from this or the other way around, please find a way, it's better code practice.

Reply:
Okay, sure.